pkg/operator/tracker.go

```diff
@@ -43,15 +43,19 @@ func (t *tprTracker) add(name, version string) {
 func (t *tprTracker) remove(name string) {
 	delete(t.clusterRVs, name)
 
+	t.Lock()
+	defer t.Unlock()
 	if stopCh, ok := t.stopChMap[name]; ok {
 		close(stopCh)
 	}
 	delete(t.stopChMap, name)
 }
 
 func (t *tprTracker) stop() {
+	t.Lock()
+	defer t.Unlock()
 	for _, stop := range t.stopChMap {
 		close(stop)
 	}
-	t.waitCluster.Wait()
+	t.stopChMap = map[string]chan struct{}{}
```

Comment:
we need to remove the channels from the map after they are closed so that a caller to `remove` or `stop` won't try to close them again. the easiest way is to create a new map